tests/integration/BUILD.bazel

```diff
@@ -67,6 +67,18 @@ py_gapic_library(
     ],
 )
 
+py_test(
```

Comment:
could we add a placeholder for asset as well?
```
# uncomment once...
#py_test(
#    name = "asset_py_gapic_test",
#    srcs = [
#        "asset_py_gapic_pytest.py",
#        "asset_py_gapic_test.py",
#    ],
#    legacy_create_init = False,
#    deps = [
#        ":asset_py_gapic",
#    ],
#)
```

Reply:
Probably. Asset has other issues though (not that much about generator but about the wierdeness of the API itself, as it depends on many other APIs)